rosdep/python.yaml

```diff
@@ -8824,6 +8824,27 @@ python3-pyqt6.qtmultimedia:
     '*': [python3-pyqt6.qtmultimedia]
     focal: null
     jammy: null
+python3-pyqt6.qtwebengine:
+  '*':
+    pip:
+      packages: [PyQt6-WebEngine]
+  debian:
+    '*': [python3-pyqt6.qtwebengine]
+    bullseye:
+      pip:
+        packages: [PyQt6-WebEngine]
+  fedora: [python3-pyqt6-webengine]
+  rhel:
+    '8': [python3.12-pyqt6-webengine]
+    '9': [python3-pyqt6-webengine]
```

Comment:
```suggestion
```

This package appears to only be available in a third party repository called Raven, not the standard EPEL repositories officially supported by ROS.